cmd/entrypoint/cmd/init.go

```diff
@@ -339,6 +345,7 @@ func init() {
 	initCmd.Flags().String(moco.PodIPFlag, "", "Pod IP address")
 	initCmd.Flags().String(moco.OperatorPasswordFlag, "", "Password for both operator user and operator admin user")
 	initCmd.Flags().String(moco.RootPasswordFlag, "", "Password for root user")
+	initCmd.Flags().String(moco.MiscPasswordFlag, "", "Password for misc user")
```

Comment:
Please tell me why you are taking passwords via command-line flags?
I'm afraid that these passwords can be leaked easily by just seeing the pod manifest.

Reply:
Actually we pass the passwords with the environment variables and don't use those command-line arguments. It's better to remove them.

Just to be clear, is it OK to pass passwords with the env variables from Secret?

Reply:
> Just to be clear, is it OK to pass passwords with the env variables from Secret?

It's okay because environment variables of others' processes can be viewed
only by the superuser.